core/mondoo-xz-vulnerability.mql.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Mondoo, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+policies:
+  - uid: mondoo-xz-vulnerability-policy
+    name: xz Vulnerability (CVE-2024–3094)
+    version: 1.0.0
+    license: BUSL-1.1
+    tags:
+      mondoo.com/category: security
+      mondoo.com/platform: linux,unix
+    authors:
+      - name: Mondoo, Inc
+        email: hello@mondoo.com
+    docs:
+      desc: |
+        The xz-utils package before 5.6.1 has a vulnerability in the xz command line tool that allows for arbitrary code execution when processing a specially crafted .xz file. This vulnerability is due to a stack-based buffer overflow in the xz_decomp function in xz-dec.c. An attacker can exploit this vulnerability by sending a specially crafted .xz file to a target system, which could allow the attacker to execute arbitrary code on the target system.
```

Comment:
We should change this wording. It's just 5.6.0 and 5.6.1. If you're on the older releases in most distros you're fine.